Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:latest
+
+RUN apt-get update \
+  && apt-get install -y software-properties-common \
+  && add-apt-repository ppa:virtuslab/git-machete \
```

Comment:
Please note that at the moment when this step is being executed, the package might still **not** be built in Ubuntu Launchpad... in fact, from my observation it takes around ~20 minutes for a package to be available in the PPA once the `ci/deb-ppa-upload/ci-deploy.sh` script is complete. The result will be that you're gonna install the **previous** version of `python3-git-machete` package, not the one you've just built in the CI for `master` branch :///

Reply:
As a remedy, I'd suggest switching to some `python3-alpine` (?) image as a base, and installing `git-machete` package from PyPI. As opposed to Ubuntu PPA (where package publishing is asynchronous), with PyPI once `ci/pypi/ci-deploy.sh` completes, the package is already available.

Reply:
In fact, to make sure that the correct version of the package is installed from PyPI, you can pass `git_machete_version` as an ARG to this dockerfile... see e.g. https://github.com/VirtusLab/git-machete/blob/master/ci/homebrew-tap/ci-deploy.sh#L13 for how to extract the version